src/backend/core/api/serializers.py

```diff
@@ -618,6 +618,12 @@ class MessageSerializer(serializers.ModelSerializer):
     )
 
     signature = ReadOnlyMessageTemplateSerializer(read_only=True, allow_null=True)
+    stmsg_headers = serializers.SerializerMethodField(read_only=True)
+
+    @extend_schema_field(serializers.DictField())
+    def get_stmsg_headers(self, instance):
```

Comment:
Shorted and more pythonic 
```suggestion
    def get_stmsg_headers(self, instance) -> dict:
```